README.md

```diff
@@ -43,6 +43,7 @@ yarn
 Now run one of the following commands:
 
 - `yarn dev` - will run development server for docs at `localhost:3000/docs` that will autorefresh pages in real time when you edit markdown documents.
+  - If you edit a partial file in `docs/pages/includes`, you will need to stop the dev server and restart it to re-include the partial.
```

Comment:
In my experience you could also make another change in the page using the partial to trigger a reload, which would then include the changes to the partial. Not sure if this works for Docker-based setups as well.

While it's certainly not an ideal solution, making a temporary arbitrary change to the parent page is probably less of a burden than restarting the server. If you don't agree, maybe we could list both options.

Reply:
I've not seen this behaviour working 100% of the time, at least in Docker. I think for now I'll leave the instructions as-is because this PR is approved. I'll submit a further fix if I find that the parent-page-edit method works. Thanks for the tip!